examples/SampleApp/fastlane/Fastfile

```diff
@@ -1,125 +1,248 @@
+    key_id: 'MT3PRT8TB7',
+    issuer_id: '69a6de96-0738-47e3-e053-5b8c7c11a4d1',
+    key_content: ENV.fetch('APPSTORE_API_KEY', nil),
+    in_house: false
+  )
+end
+
+desc "If `readonly: true` (by default), installs all Certs and Profiles necessary for development and ad-hoc.\nIf `readonly: false`, recreates all Profiles necessary for development and ad-hoc, updates them locally and remotely."
+lane :match_me do |options|
+  custom_match(
+    api_key: appstore_api_key,
+    app_identifier: [bundle_id],
+    readonly: options[:readonly],
+    register_device: options[:register_device]
+  )
+end
 
-      match_appstore
+##########################
+###### Common lanes ######
+##########################
 
-      settings_to_override = {
-        :BUNDLE_IDENTIFIER => "io.getstream.reactnative.SampleApp",
-        :PROVISIONING_PROFILE_SPECIFIER => "match AppStore io.getstream.reactnative.SampleApp"
-      }
+lane :file_size do |options|
+  File.size(options[:path]).to_f / 1024
+end
 
-      package = load_json(json_path: "./package.json")
-      increment_version_number(version_number: package["version"], xcodeproj: "./ios/SampleApp.xcodeproj")
-      current_build_number = app_store_build_number(
-        live: false,
-        app_identifier: "io.getstream.reactnative.SampleApp",
-      )
-      increment_build_number(
-        build_number: current_build_number + 1,
-        xcodeproj: "./ios/SampleApp.xcodeproj"
-      )
+lane :directory_size do |options|
+  total_size = 0
+  Find.find(options[:path]) do |file|
+    total_size += File.size(file) if File.file?(file)
+  end
+  total_size.to_f / 1024
+end
 
-      gym(
-          workspace: "./ios/SampleApp.xcworkspace",
-          scheme: "SampleApp",
-          export_method: "app-store",
-          export_options: "./fastlane/testflight_gym_export_options.plist",
-          silent: true,
-          clean: true,
-          xcargs: settings_to_override,
-          include_symbols: true,
-          output_directory: "./dist",
-      )
+lane :create_template_project do
+  Dir.chdir(temp_dir) do
+    sh("npx @react-native-community/cli@latest init #{template_app_name} --install-pods false --package-name #{bundle_id}")
+    File.absolute_path(template_app_name)
+  end
+end
 
-      if options[:deploy]
-        begin
-          upload_to_testflight(
-              groups: ['Testers'],
-              changelog: "Lots of amazing new features to test out!",
-              reject_build_waiting_for_review: false
-          )
-        rescue Exception => e
-          if e.message.include? "Another build is in review"
-            UI.important("Another build is already in beta review. Skipping beta review submission")
-          else
-            raise
-          end
-        end
-      end
+lane :yarn_sdk do |options|
+  Dir.chdir(options[:path]) do
+    node_modules = [
+      # FIXME: "stream-chat-react-native@https://github.com/GetStream/stream-chat-react-native.git##{current_branch}"
```

Comment:
⚠️ We still need to somehow implement the SDK installation from the current branch.